cypress/e2e/tests/migration/controls/businessservices/sort.test.ts

```diff
@@ -76,15 +59,15 @@ describe(["@tier2"], "Business services sort validations", function () {
         const unsortedList = getTableColumnData(name);
 
         // Sort the business services by name in ascending order
-        sortAsc(name);
+        clickOnSortButton(name, "ascending", businessServiceTable);
         cy.wait(2000);
 
         // Verify that the business services table rows are displayed in ascending order
         const afterAscSortList = getTableColumnData(name);
         verifySortAsc(afterAscSortList, unsortedList);
 
         // Sort the business services by name in descending order
-        sortDesc(name);
+        clickOnSortButton(name, "descending", businessServiceTable);
```

Comment:
**nitpick:** `ascending` and `descending` are words that can be mistyped very easily, I'd prefer to use two constants or an enum to avoid errors, but that's a matter of personal taste.

Reply:
You are right , I thought about it but got lazy :). Will fix it .